docs/developers/README.md

```diff
@@ -155,3 +156,24 @@ such as the following:
 - The environment variable HOME is set to `/builder/home`, used by the builder
   tools and injected on into all of the step containers
 - Default location for output-images `/builder/output-images`
+
+## Handling of injected sidecars
+
+The [Downward API](TODO: link) is used to project an annotation on the
```

Comment:
https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api seems okay but I tried to find docs that actually really explain how this works and i couldn't find anything very good :(